scene/gui/rich_text_label.cpp

```diff
@@ -29,6 +29,7 @@
 /*************************************************************************/
 
 #include "rich_text_label.h"
+#include "../../modules/modules_enabled.gen.h"
```

Comment:
This should be below together with other includes. It should also start from the root, we don't use relative paths.
`modules/regex/regex.h` should not be included if the module is disabled.